sdk/src/fee_calculator.rs

```diff
@@ -67,7 +67,6 @@ impl FeeCalculator {
 pub struct FeeRateGovernor {
     // The current cost of a signature  This amount may increase/decrease over time based on
     // cluster processing load.
-    #[serde(skip)]
```

Comment:
this is where gating would be necessary. Alternatively we just set this value on deserialize to `bank.fee_calculator.lamports_per_signature`

Reply:
@ryoqun after talking with @t-nelson, it seems removing the skip here directly also breaks RPC, seems like the path of least resistance is to just manually assign this field for now. Thoughts?

Reply:
> the path of least resistance is to just manually assign this field for now. Thoughts?

@carllin yeah, I think this solution is good.

Reply:
Also, when did this bug started to occur? Since beginning? ;)

Reply:
hmmm yeah I would guess since it was first checked in xD